addons/website/static/src/scss/website.ui.scss

```diff
@@ -74,6 +74,12 @@ $-mini-nav-size: 40px;
             transition-duration: 400ms;
         }
     }
+
+    @include media-breakpoint-down(md) {
+        &:not(.d-none) ~ .o_thread_window .o_thread_window_title{
```

Comment:
This doesn't exists in website, to be moved in website_livechat?

Reply:
I'll put it in `im_livechat` since that's where all the styles for `.o_thread_window` are already.
Thanks for checking!